libraries/AP_Mount/AP_Mount_Siyi.h

```diff
@@ -136,6 +136,12 @@ class AP_Mount_Siyi : public AP_Mount_Backend
         ZR10
     } _hardware_model;
 
+    // gimbal mounting method/direction
+    enum class GimbalMountingDirection : uint8_t {
+        NORMAL = 1,
```

Comment:
As mentioned in Discord, let' change this enum to start from zero so that it's "normal" on startup.

Reply:
@rmackay9 , The 0 is reserved. I think What we should do is start from Reserved:0 than normal and upsideDown. However reserved will not be used.